iio-widgets/src/datastrategy/cmdqdeviceattrdatastrategy.cpp

```diff
@@ -111,6 +116,9 @@ void CmdQDeviceAttrDataStrategy::attributeReadFinished(Command *cmd)
 		m_cmdQueue->enqueue(readOptionalCommand);
```

Comment:
The same problem as here:  https://github.com/analogdevicesinc/scopy/pull/1601/files#r1593514496